### DISCUSSION TITLE
Box Art Downloader/Scraper?

### DISCUSSION THREAD

**Discussion by DaveH-Infra** [OnionUI/Onion#759] [Ideas]
Hi, with the Miyoo Mini Plus having wifi, would there be any way to have an option to automatically scrape or download box art for games on the Miyoo Mini handheld rather than using a computer?

Obviously trying to do every game at once will be a bit much for the system, but maybe if you click the menu button when browsing games you can have an option to find box art?

Thanks

**Comment by Fr33hk** [comment]
Absolutely second this request, I'm assuming there's a Linux based solution already in place as this is accomplished on the raspberry pi

**Comment by schmurtzm** [comment]
It’s in the roadmap 😉 there is a issue for it.
https://github.com/OnionUI/Onion/issues/771

**Comment by schmurtzm** [comment]
Closing it.
Supported [since Onion 4.2.0.](https://onionui.github.io/docs/apps/scraper)
Region settings, Media types, CRC search... implemented since PR https://github.com/OnionUI/Onion/pull/1110
